docs/pages/versions/v32.0.0/guides/using-clojurescript.md

```diff
@@ -134,7 +134,7 @@ React Native uses JavascriptCore, so modules using built-in node like stream, fs
 
 ### Do I need to restart the REPL after adding new Javascript modules or assets?
 
-No, you do need to reload Javascript. To do that, select **Reload** from the Developer Menu. You can also press `⌘ + R` in the iOS Simulator, or press `R` twice on Android emulators.
+Yes, you do need to reload Javascript. To do that, select **Reload** from the Developer Menu. You can also press `⌘ + R` in the iOS Simulator, or press `R` twice on Android emulators.
```

Comment:
This is still ambiguous and the rest of the instructions don't refer to restarting the REPL -> need to clarify further.